dusty/daemon.py

```diff
@@ -42,6 +42,12 @@ def _refresh_warnings():
         refresh_config_warnings()
         refresh_preflight_warnings()
 
+def _run_pre_command_functions(connection, suppress_warnings, client_version):
+    _refresh_warnings()
+    _send_warnings_to_client(connection, suppress_warnings)
```

Comment:
Should this line also not be grouped with this function? I feel like we have different ideas of what actually belongs here